client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useSelect } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
+import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
+import { useInterval } from 'calypso/lib/interval';
+import type { Step } from '../../types';
+import 'calypso/assets/stylesheets/shared/_loading.scss';
+import './style.scss';
+
+const useSteps = () => {
+	const { __ } = useI18n();
+	const steps = [
+		{ title: __( 'Laying the foundations' ), duration: 2000 },
+		{ title: __( 'Turning on the lights' ), duration: 3000 },
+		{ title: __( 'Making it beautiful' ), duration: 2000 },
+		{ title: __( 'Personalizing your site' ), duration: 4000 },
+		{ title: __( 'Sprinkling some magic' ), duration: 4000 },
+		{ title: __( 'Securing your data' ), duration: 5000 },
+		{ title: __( 'Enabling encryption' ), duration: 3000 },
+		{ title: __( 'Optimizing your content' ), duration: 6000 },
+		{ title: __( 'Applying a shiny top coat' ), duration: 4000 },
+		{ title: __( 'Closing the loop' ), duration: 5000 },
+	];
+
+	return useRef( steps.filter( Boolean ) );
```

Comment:
I have two questions:

1. Does this filter filter anything?
2. Is the ref needed? It isn't modified anywhere AFAICT.